tests/TestDateTime.elm

```diff
@@ -235,7 +235,7 @@ toFromISO8601 =
             [ test "toISO8601 of epoch is correct" <|
                 \() ->
                     toISO8601 epoch
-                        |> Expect.equal "1970-01-01T00:00:00Z"
+                        |> Expect.equal "1970-01-01T00:00:00.000Z"
```

Comment:
This is how `date.toISOString()` works in JS, so I decided not to cut the millis when zero.